tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/store/debugger_selectors.ts

```diff
@@ -68,6 +70,49 @@ export const getNumAlerts = createSelector(
   }
 );
 
+export const getAlertsFocusType = createSelector(
+  selectDebuggerState,
+  (state: DebuggerState): AlertType | null => {
+    return state.alerts.focusType;
+  }
+);
+
+/**
+ * Get number of alerts of the alert type being focused on.
```

Comment:
For future changes, JSDocs tend to use verb in third person (i.e., `Gets`).

Reply:
https://google.github.io/styleguide/jsguide.html#jsdoc-method-and-function-comments

The relevant part: 

> Method descriptions begin with a verb phrase that describes what the method does. This phrase is not an imperative sentence, but instead is written in the third person, as if there is an implied This method ... before it.